temporal-sdk/src/main/java/io/temporal/internal/worker/LocalActivityWorker.java

```diff
@@ -93,6 +93,10 @@ public void start() {
     }
   }
 
+  public boolean hasSupportedTypes() {
```

Comment:
Any reason for using a different name?

Reply:
If you feel strongly about the new name then rename the one on the handler to be consistent.

Reply:
No strong opinion, it felt a bit more natural to me this way, but I can totally see consistency argument. Let me rename it.